editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/importActions/ImportActions.tsx

```diff
@@ -65,6 +67,17 @@ export default function ImportActions(props: {
 				<span className="icon icon-refresh mr-1 pos-adjusted-down"></span>
 				Sync ({props.importedItemsCount})
 			</button>
+			<button
+				className="button button-secondary button-small flex flex-centered"
+				disabled={props.relationsCount == 0}
+				data-test="relations"
+				onClick={() => props.onRelations()}
```

Comment:
Lacks feedback, becoming ever more apparent for larger amounts of data. The button itself should display a spinner at least.

![assoc_rel_loading_feedback](https://github.com/codeBeamer-Extensions-and-Addons/codebeamer-lucidspark/assets/70019423/2ebf441d-08a8-44ee-a724-0a8133dbcdfb)